sources/utils/googleCalendarUtils.ts

```diff
@@ -0,0 +1,81 @@
+import { google, calendar_v3 } from "googleapis";
+import { GetTokenResponse, OAuth2Client } from "google-auth-library/build/src/auth/oauth2client";
+
+import { GoogleCalendarCredentials } from "../model/entity/GoogleCalendarCredentials.js";
+import { Calendar } from "../model/entity/Calendar.js";
+import { Event } from "../model/entity/Event.js";
+import { ApiException } from "./apiException.js";
+
+const oauth2Client = new google.auth.OAuth2(
+    process.env.CLIENT_ID,
+    process.env.CIENT_SECRET,
+    process.env.API_HOST + "/calendars/import-google-calendar/",
+);
+
+export class GoogleCalendarUtils {
```

Comment:
Consider renaming the file following `PascalCase` since it contains a class. `camelCase` is for file containing only functions